test/index/validator/server.test.ts

```diff
@@ -91,7 +91,7 @@ describe('index/validator/server', () => {
 
 				// When
 				// Then
-				expect(new ServerValidator(options)).to.eql(options);
+				expect(new ServerValidator(options)).to.not.throw;
```

Comment:
The expected behaviour seems a little vague? Might be better to check something like:
```
// When
const validator = new ServerValidator(options);

// Then
expect(validator).to.be.an.instanceof(CommonValidator);
```

Reply:
If the options are valid then the validator should not throw an error.

I've added another test to make sure that they extend CommonValidator.